crates/uv/tests/cache_clean.rs

```diff
@@ -0,0 +1,143 @@
+#![cfg(all(feature = "python", feature = "pypi"))]
+
+use std::process::Command;
+
+use anyhow::Result;
+use assert_cmd::prelude::*;
+use assert_fs::prelude::*;
+
+use common::uv_snapshot;
+
+use crate::common::{get_bin, TestContext};
+
+mod common;
+
+/// Create a `cache clean` command with options shared across scenarios.
+fn clean_command(context: &TestContext) -> Command {
+    let mut command = Command::new(get_bin());
+    command.arg("cache").arg("clean");
+    context.add_shared_args(&mut command);
+    command
+}
```

Comment:
Technically this is bad form, can you add it to the `TestContext` to match the other commands?

Reply:
Yup.

Reply:
(I cribbed this from `cache_prune.rs` so changing that too.)

Reply:
Thanks!